typescript/infra/config/environments/mainnet3/governance/ica/regular.ts

```diff
@@ -2,110 +2,109 @@ import { ChainMap } from '@hyperlane-xyz/sdk';
-  // metis: '0x319aB070390C8f2A914a61f8DE366f3295bb44fF',
-  // mint: '0x319aB070390C8f2A914a61f8DE366f3295bb44fF',
-  // molten: '0x100D940B953b20378242Fe0FbC45792805d50556',
-  // moonbeam: '0xb3178E470E959340206AD9d9eE4E53d0911b2ba9',
-  // morph: '0x83be90021870A93ff794E00E8CFe454547877E3E',
-  // nero: '0x4601d6260C532E73187248D0608cB88D446149AD',
-  // nibiru: '0x966A57078204CA6cA13f091eE66eE61Ada5D2318',
-  // oortmainnet: '0xBc2cc7372C018C2a0b1F78E29D8B49d21bc5f3bA',
-  // opbnb: '0x1fbE174fc6B1d3123B890f34aBd982577377Bec8',
-  // orderly: '0x83be90021870A93ff794E00E8CFe454547877E3E',
-  // plume: '0xB9d20ea5f3bB574C923F0af67b06b8D87F111819',
-  // polygon: '0xff5b2F0Ae1FFBf6b92Ea9B5851D7643C81102064',
-  // polygonzkevm: '0x258B14038C610f7889C74162760A3A70Ce526CD5',
-  // polynomialfi: '0x45EFBEAD9009C2662a633ee6F307C90B647B7793',
-  // prom: '0x9123625687807F02FC92A9817cFb8db13A9a8B4d',
-  // proofofplay: '0x319aB070390C8f2A914a61f8DE366f3295bb44fF',
-  // rarichain: '0x80c7c5fF8F84BEeDEd18382eb45aA3246063fbc5',
-  // reactive: '0x97B3BE8f063E98Da6ac254716DA194da4634aC80',
-  // real: '0xdc13fF36978e860d2FC98EaFBA4fDeB64E1D864A',
-  // redstone: '0x27C06C13E512Cdd80A2E49fc9c803cFd0de7Ba9e',
-  // rivalz: '0xBd887119d776f0c990e9a03B1A157A107CD45033',
-  // ronin: '0xa3EC6913675e9686bfC458F02a0F737EdD0362c8',
-  // rootstockmainnet: '0x77d190423A18e78C95269f4D1349141D15E433f3',
-  // sanko: '0x27C06C13E512Cdd80A2E49fc9c803cFd0de7Ba9e',
-  // scroll: '0xDD2e306EE337952b4f7C0c4Eb44A8B5b9925Bc76',
-  // shibarium: '0x778f0e4CD1434258A811ab493217f3DC8d501C1b',
-  // snaxchain: '0x83be90021870A93ff794E00E8CFe454547877E3E',
-  // soneium: '0x4BAe702C0D1cC7d6d343594735812b5723041528',
-  // sonic: '0x4BAe702C0D1cC7d6d343594735812b5723041528',
-  // story: '0xa3EC6913675e9686bfC458F02a0F737EdD0362c8',
-  // subtensor: '0xB9d20ea5f3bB574C923F0af67b06b8D87F111819',
-  // superseed: '0xEc65c06c606006Db682197230bF8E7740C2BcFF9',
-  // superpositionmainnet: '0x9f921e87309829ecc61F6df0F2016fD08B80CDD1',
-  // tangle: '0x60109e724a30E89fEB1877fAd006317f8b81768a',
-  // telos: '0xf36dC13eE34034709390b6dF3de7305eA298BFec',
-  // torus: '0xBd887119d776f0c990e9a03B1A157A107CD45033',
-  // trumpchain: '0x3FAA6Ff31e4FC67957792303bA35092Ee0EBd367',
-  // unichain: '0xEc65c06c606006Db682197230bF8E7740C2BcFF9',
-  // unitzero: '0x262E9A3eDA180fCb84846BaBa2aEB9EDa88e9BeF',
-  // vana: '0xEc65c06c606006Db682197230bF8E7740C2BcFF9',
-  // viction: '0x426FC4C5CC60E5e47101fe30d4f8B94F1b7C1C70',
-  // worldchain: '0x51E4E4b7317043A1D6e2fe6ccd9dA51844C385f0',
-  // xai: '0xF01B503E5cC884f0EB6eF062F704A610FD69842D',
-  // xlayer: '0x73581C1c03E64BDE5775b7236B57d78155B75fA7',
-  // xpla: '0x0Fb0635eAbB9332eDc2544A57e7C15fBc5204C0B',
-  // zetachain: '0x33d8605D4dF6661259640268DD32BC99A2F55a83',
-  // zoramainnet: '0xdbB6E8FDe96dE3a1604b93884d8fF9Cb521059fC',
+  ancient8: '0xf789D8a609247c448E28d3af5b8EFC6Cb786C4ee',
```

Comment:
What kind of process and/or verification has gone into this?

Reply:
This file was populated by running the get-owner-ica script. It was checked by:
1. running the check-owner-ica script for `--governanceType regular`
2. as part of liveness testing @Mo-Hussain also proposed remote ICA calls to do spot checks that we are able to successfully make calls from the origin safe to these remote ICAs

Reply:
Got it. Just spot checked a few as well